.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "go.testEnvVars": {
+        "CONFIG_PATH": "${workspaceFolder}/configs"
```

Comment:
Would "./configs" not work here?

For the .vscode folder to work I thought you had to have the project folder open anyway.

Reply:
Sorry for the delay mate! I was getting an error when launching the tests in visual studio code; by the situation that I got and the observations, from the ide the test binaries are generated in each package directory; when the tests are run from the ide, the current working directory is changed to the package directory, making the relative reference wrong.

It happens when we try to debug a test from the test lab view. I attach a screenshot

Reply:
![image](https://user-images.githubusercontent.com/3994774/175997650-ca257eb5-1562-4180-847a-db274de42932.png)